### DISCUSSION TITLE
Serial or parallel Execution

### DISCUSSION THREAD

**Discussion by almondelm** [olahallengren/sql-server-maintenance-solution#418] [Questions & Answers]
Hello Ola

Thanks for helping out. Such a neat solution which is great help for me. This is not a issue , just general question.
I am wondering if your index maintenance and statistics jobs are serial or parallel execution ? Looking for some ways to parallel execution of our stats maintenance to cut down time during maintenance window.
Appreciate your help.
 
Thanks and take care

Sandip

**Comment by olahallengren** [comment]
>I am wondering if your index maintenance and statistics jobs are serial or parallel execution ?

It is processing the indexes within a database serially.

I have support for processing multiple databases in parallel.
